docs/hassio_debugging.md

```diff
@@ -45,4 +39,20 @@ docker logs hassos_supervisor
 docker logs homeassistant
 ```
 
+## Accessing the container bash
```

Comment:
This section is kinda weird...
You are already in bash when reaching this point...?!

Reply:
you're in bash on the host, not in the homeassistant container.

Reply:
The page is about accessing the host system... 
you made it about accessing HA in the running HA container via the host system 😉

Reply:
Yeah, it's a little confusing.  I updated the page because it didn't spell out that you needed to run the login command from CLI to get to the host (even though the CLI welcome text says it).  The title of the page is "Debugging Hass.io" but right now it mainly just explains that you need to setup SSH keys and how to access logs.  It doesn't try to explain the differences between Hassio CLI, host bash and homeassistant container bash (maybe it just assumes developers should already know that).  Case in point, earlier I was trying to debug what I thought might be an ffmpeg issue and you can't debug that from Hassio CLI or host bash, you need to go to the container bash to do it.

Reply:
> "just assumes developers should already know that"

In that case, we could just as well assume a developer knows how to access Docker containers?

> "Yeah, it's a little confusing."

Could you please address it in this PR? 👍